Gems/Maestro/Code/Source/Cinematics/AnimPostFXNode.cpp

```diff
@@ -38,7 +38,7 @@ class CFXNodeDescription
         virtual void GetDefault(bool& val) const = 0;
         virtual void GetDefault(Vec4& val) const = 0;
 
-        AZStd::string m_name;
+        std::string m_name;
```

Comment:
What is the motivation for this change?

Reply:
The commit message for commit 6a9430533 has the details:

>This code uses global static instances, which have a nondeterministic
initialization order. Due to the fact that they are global, the AZ
allocators are not guaranteed to be initialized first. To deal with
this, most of this code already uses `std` containers over `AZStd` ones.
It stores a `CAnimPostFXNode::FxNodeDescriptionMap`, which is a
`std::map`, the values of which are `CFXNodeDescription`s. Those
descriptions were storing an `AZStd::string`. Putting an `AZStd::string`
in a global static variable is not guaranteed to work.